_posts/0000-01-26-Jamila-Bay.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome to my GitHub!
```

Comment:
Test writing a comment

Reply:
Replying to last comment from cherylrCode